LICENSE.md

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015 [these people](https://github.com/sebs/es6-fuzz/graphs/contributors)
+Copyright (c) 2015 [these people](https://github.com/skanderturki/js-fuzzy-logic/graphs/contributors)
```

Comment:
I think the link in my repo represents very well who contributed.

Reply:
ok, I'm not used to this, thanks

Reply:
no problem ;)